docs/diga_api.md

```diff
@@ -0,0 +1,216 @@
+# Overview
+
+Manufacturers of digitial health applications (DiGA) can apply for a fast track at [bfarm](https://www.bfarm.de/EN/MedicalDevices/DiGA/_node.html) to place their solutions in a directory of apps which can be prescribed by doctors to patients. By showing evidence that the apps improve patients health and by obtaining medical device certifications, it can be assured that the apps are of high quality. On the other side, DiGA manufacturers get reimbursed by health insurances when their apps are prescribed to patients. The prescription process is build around a 16-character prescription code which can be used to activate the apps but which is also required to send the invoices. This repository holds code to:
```

Comment:
`patients'`

process is buil**t**

Reply:
Done. Also added linbreaks after sentences. Makes it easier to add comments and find the spot comment refer to